### ISSUE TITLE
[BUG] Auth Setup tests crash browser when ran in headed mode

### ISSUE THREAD

**Issue by deepanchal** [microsoft/playwright#24289]
### System info
- Playwright Version: [v1.36.1]
- Operating System: [Linux 6.4.2-3-MANJARO]
- Browser: [All]
- Other info:

### Source code

- [x] I provided exact source code that allows reproducing the issue locally.

**Link to the GitHub repository with the repro**

[[https://github.com/deepanchal/playwright-auth-setup-tests-bug]](https://github.com/deepanchal/playwright-auth-setup-tests-bug)

**Steps**

- Clone the [repo](https://github.com/deepanchal/playwright-auth-setup-tests-bug)
- `cd playwright-auth-setup-tests-bug`
- `npm install`
- `npx playwright install`
- `npm run test:e2e` - Runs `playwright test` - Everything passes in headless mode
- `npm run test:e2e:headed` - Runs `playwright test --headed` - Browser opens and then crashes when running auth setup tests and skips all other tests

**Logs**

Output from `DEBUG='pw:api,pw:browser*' pnpm test:e2e:headed`

```log
Running 4 tests using 2 workers
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:api => selectors.setTestIdAttribute started +0ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:api => selectors.setTestIdAttribute started +0ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:api => browserType.launch started +4ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:api => browserType.launch started +4ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:api <= selectors.setTestIdAttribute succeeded +6ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:api <= selectors.setTestIdAttribute succeeded +6ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser <launching> /home/deep/.cache/ms-playwright/chromium-1071/chrome-linux/chrome --disable-field-trial-config --disable-background-networking --enable-features=NetworkService,NetworkServiceInProcess --disable-background-timer-throttling --disable-backgrounding-occluded-windows --disable-back-forward-cache --disable-breakpad --disable-client-side-phishing-detection --disable-component-extensions-with-background-pages --disable-component-update --no-default-browser-check --disable-default-apps --disable-dev-shm-usage --disable-extensions --disable-features=ImprovedCookieControls,LazyFrameLoading,GlobalMediaControls,DestroyProfileOnBrowserClose,MediaRouter,DialMediaRouteProvider,AcceptCHFrame,AutoExpandDetailsElement,CertificateTransparencyComponentUpdater,AvoidUnnecessaryBeforeUnloadCheckSync,Translate --allow-pre-commit-input --disable-hang-monitor --disable-ipc-flooding-protection --disable-popup-blocking --disable-prompt-on-repost --disable-renderer-backgrounding --force-color-profile=srgb --metrics-recording-only --no-first-run --enable-automation --password-store=basic --use-mock-keychain --no-service-autorun --export-tagged-pdf --no-sandbox --user-data-dir=/tmp/playwright_chromiumdev_profile-5I2MQ2 --remote-debugging-pipe --no-startup-window +0ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser <launching> /home/deep/.cache/ms-playwright/chromium-1071/chrome-linux/chrome --disable-field-trial-config --disable-background-networking --enable-features=NetworkService,NetworkServiceInProcess --disable-background-timer-throttling --disable-backgrounding-occluded-windows --disable-back-forward-cache --disable-breakpad --disable-client-side-phishing-detection --disable-component-extensions-with-background-pages --disable-component-update --no-default-browser-check --disable-default-apps --disable-dev-shm-usage --disable-extensions --disable-features=ImprovedCookieControls,LazyFrameLoading,GlobalMediaControls,DestroyProfileOnBrowserClose,MediaRouter,DialMediaRouteProvider,AcceptCHFrame,AutoExpandDetailsElement,CertificateTransparencyComponentUpdater,AvoidUnnecessaryBeforeUnloadCheckSync,Translate --allow-pre-commit-input --disable-hang-monitor --disable-ipc-flooding-protection --disable-popup-blocking --disable-prompt-on-repost --disable-renderer-backgrounding --force-color-profile=srgb --metrics-recording-only --no-first-run --enable-automation --password-store=basic --use-mock-keychain --no-service-autorun --export-tagged-pdf --no-sandbox --user-data-dir=/tmp/playwright_chromiumdev_profile-G9J9QP --remote-debugging-pipe --no-startup-window +0ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser <launched> pid=794121 +5ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser <launched> pid=794122 +3ms
  pw:browser [pid=794122][err] Gtk-Message: 14:09:41.482: Failed to load module "appmenu-gtk-module" +96ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser [pid=794121][err] Gtk-Message: 14:09:41.482: Failed to load module "appmenu-gtk-module" +97ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser [pid=794122][err] [794122:794122:0718/140941.552875:ERROR:chrome_browser_cloud_management_controller.cc(162)] Cloud management controller initialization aborted as CBCM is not enabled. +70ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser [pid=794121][err] [794121:794121:0718/140941.579003:ERROR:chrome_browser_cloud_management_controller.cc(162)] Cloud management controller initialization aborted as CBCM is not enabled. +96ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:api <= browserType.launch succeeded +272ms
  pw:api => browser.newContext started +4ms
  pw:api <= browser.newContext succeeded +11ms
  pw:api => browserContext.newPage started +6ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:api <= browserType.launch succeeded +295ms
  pw:api => browser.newContext started +4ms
  pw:api <= browser.newContext succeeded +8ms
  pw:api => browserContext.newPage started +5ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser [pid=794122][err] [0718/140941.651295:ERROR:elf_dynamic_array_reader.h(64)] tag not found +98ms
  pw:browser [pid=794122][err] [0718/140941.651647:ERROR:elf_dynamic_array_reader.h(64)] tag not found +0ms
  pw:browser [pid=794122][err] [0718/140941.651706:ERROR:elf_dynamic_array_reader.h(64)] tag not found +0ms
  pw:browser [pid=794122][err] [0718/140941.653636:ERROR:file_io_posix.cc(144)] open /sys/devices/system/cpu/cpu0/cpufreq/scaling_cur_freq: No such file or directory (2) +2ms
  pw:browser [pid=794122][err] [0718/140941.653652:ERROR:file_io_posix.cc(144)] open /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq: No such file or directory (2) +0ms
  pw:browser [pid=794122][err] Received signal 11 SEGV_MAPERR 000000000148 +5ms
  pw:browser [pid=794122][err] #0 0x55d34825d302 base::debug::CollectStackTrace() +5ms
  pw:browser [pid=794122][err] #1 0x55d34824a753 base::debug::StackTrace::StackTrace() +4ms
  pw:browser [pid=794122][err] #2 0x55d34825cd91 base::debug::(anonymous namespace)::StackDumpSignalHandler() +3ms
  pw:browser [pid=794122][err] #3 0x7f3c5b4bfab0 (/usr/lib/libc.so.6+0x39aaf) +1ms
  pw:browser [pid=794122][err] #4 0x55d343fc9670 base::ObserverList<>::AddObserver() +0ms
  pw:browser [pid=794122][err] #5 0x55d34da99375 AvatarMenu::AvatarMenu() +14ms
  pw:browser [pid=794122][err] #6 0x55d34da95dff DbusAppmenu::Initialize() +12ms
  pw:browser [pid=794122][err] #7 0x55d34da98296 DbusAppmenuRegistrar::InitializeMenu() +10ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser [pid=794121][err] [0718/140941.708612:ERROR:elf_dynamic_array_reader.h(64)] tag not found +129ms
  pw:browser [pid=794121][err] [0718/140941.709143:ERROR:elf_dynamic_array_reader.h(64)] tag not found +1ms
  pw:browser [pid=794121][err] [0718/140941.709227:ERROR:elf_dynamic_array_reader.h(64)] tag not found +0ms
  pw:browser [pid=794121][err] [0718/140941.711820:ERROR:file_io_posix.cc(144)] open /sys/devices/system/cpu/cpu0/cpufreq/scaling_cur_freq: No such file or directory (2) +2ms
  pw:browser [pid=794121][err] [0718/140941.711844:ERROR:file_io_posix.cc(144)] open /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq: No such file or directory (2) +1ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser [pid=794122][err] #8 0x55d34da98aa3 DbusAppmenuRegistrar::OnNameOwnerChanged() +9ms
  pw:browser [pid=794122][err] #9 0x55d343fbc4f1 base::internal::Invoker<>::Run() +0ms
  pw:browser [pid=794122][err] #10 0x55d3447b0911 base::internal::Invoker<>::RunOnce() +1ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser [pid=794121][err] Received signal 11 SEGV_MAPERR 000000000148 +5ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser [pid=794122][err] #11 0x55d3481ed1c4 base::TaskAnnotator::RunTaskImpl() +4ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser [pid=794121][err] #0 0x55b529df1302 base::debug::CollectStackTrace() +6ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser [pid=794122][err] #12 0x55d34820752f base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWorkImpl() +4ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser [pid=794121][err] #1 0x55b529dde753 base::debug::StackTrace::StackTrace() +4ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser [pid=794122][err] #13 0x55d348206fa9 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWork() +4ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser [pid=794121][err] #2 0x55b529df0d91 base::debug::(anonymous namespace)::StackDumpSignalHandler() +4ms
  pw:browser [pid=794121][err] #3 0x7fe52acf1ab0 (/usr/lib/libc.so.6+0x39aaf) +0ms
  pw:browser [pid=794121][err] #4 0x55b525b5d670 base::ObserverList<>::AddObserver() +0ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser [pid=794122][err] #14 0x55d348207a05 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWork() +4ms
  pw:browser [pid=794122][err] #15 0x55d348270e14 base::MessagePumpGlib::Run() +4ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser [pid=794121][err] #5 0x55b52f62d375 AvatarMenu::AvatarMenu() +10ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser [pid=794122][err] #16 0x55d348207d79 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::Run() +6ms
  pw:browser [pid=794122][err] #17 0x55d3481cde66 base::RunLoop::Run() +5ms
  pw:browser [pid=794122][err] #18 0x55d3461b0fa3 content::BrowserMainLoop::RunMainMessageLoop() +2ms
  pw:browser [pid=794122][err] #19 0x55d3461b2aa2 content::BrowserMainRunnerImpl::Run() +2ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser [pid=794121][err] #6 0x55b52f629dff DbusAppmenu::Initialize() +12ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser [pid=794122][err] #20 0x55d3461ae616 content::BrowserMain() +2ms
  pw:browser [pid=794122][err] #21 0x55d347653088 content::RunBrowserProcessMain() +3ms
  pw:browser [pid=794122][err] #22 0x55d3476549e4 content::ContentMainRunnerImpl::RunBrowser() +3ms
  pw:browser [pid=794122][err] #23 0x55d3476547d6 content::ContentMainRunnerImpl::Run() +3ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser [pid=794121][err] #7 0x55b52f62c296 DbusAppmenuRegistrar::InitializeMenu() +11ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser [pid=794122][err] #24 0x55d347651d90 content::RunContentProcess() +4ms
  pw:browser [pid=794122][err] #25 0x55d347651fbd content::ContentMain() +3ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser [pid=794121][err] #8 0x55b52f62caa3 DbusAppmenuRegistrar::OnNameOwnerChanged() +13ms
  pw:browser [pid=794121][err] #9 0x55b525b504f1 base::internal::Invoker<>::Run() +0ms
  pw:browser [pid=794121][err] #10 0x55b526344911 base::internal::Invoker<>::RunOnce() +0ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser [pid=794122][err] #26 0x55d343e1a309 ChromeMain +11ms
  pw:browser [pid=794122][err] #27 0x7f3c5b4a9850 (/usr/lib/libc.so.6+0x2384f) +1ms
  pw:browser [pid=794122][err] #28 0x7f3c5b4a990a __libc_start_main +0ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser [pid=794121][err] #11 0x55b529d811c4 base::TaskAnnotator::RunTaskImpl() +6ms
  pw:browser [pid=794121][err] #12 0x55b529d9b52f base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWorkImpl() +5ms
  pw:browser [pid=794121][err] #13 0x55b529d9afa9 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWork() +4ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser [pid=794122][err] #29 0x55d343e1a02a _start +11ms
  pw:browser [pid=794122][err]   r8: 0000000000000002  r9: 0000000000000000 r10: 000055d34ec43b80 r11: 0000000000000040 +0ms
  pw:browser [pid=794122][err]  r12: 000055d34eb8a000 r13: 00003a4800f32d40 r14: 00003a4800f32d00 r15: 00003a4800f32d18 +0ms
  pw:browser [pid=794122][err]   di: 0000000000000148  si: 00003a4800f32d00  bp: 00007ffe29b90130  bx: 0000000000000148 +0ms
  pw:browser [pid=794122][err]   dx: 000055d341f38b20  ax: 0000000000000000  cx: 000000004e593401  sp: 00007ffe29b900e0 +0ms
  pw:browser [pid=794122][err]   ip: 000055d343fc9670 efl: 0000000000010206 cgf: 002b000000000033 erf: 0000000000000004 +0ms
  pw:browser [pid=794122][err]  trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000148 +0ms
  pw:browser [pid=794122][err] [end of stack trace] +0ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser [pid=794121][err] #14 0x55b529d9ba05 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWork() +5ms
  pw:browser [pid=794121][err] #15 0x55b529e04e14 base::MessagePumpGlib::Run() +4ms
  pw:browser [pid=794121][err] #16 0x55b529d9bd79 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::Run() +4ms
  pw:browser [pid=794121][err] #17 0x55b529d61e66 base::RunLoop::Run() +4ms
  pw:browser [pid=794121][err] #18 0x55b527d44fa3 content::BrowserMainLoop::RunMainMessageLoop() +2ms
  pw:browser [pid=794121][err] #19 0x55b527d46aa2 content::BrowserMainRunnerImpl::Run() +2ms
  pw:browser [pid=794121][err] #20 0x55b527d42616 content::BrowserMain() +2ms
  pw:browser [pid=794121][err] #21 0x55b5291e7088 content::RunBrowserProcessMain() +3ms
  pw:browser [pid=794121][err] #22 0x55b5291e89e4 content::ContentMainRunnerImpl::RunBrowser() +3ms
  pw:browser [pid=794121][err] #23 0x55b5291e87d6 content::ContentMainRunnerImpl::Run() +3ms
  pw:browser [pid=794121][err] #24 0x55b5291e5d90 content::RunContentProcess() +4ms
  pw:browser [pid=794121][err] #25 0x55b5291e5fbd content::ContentMain() +3ms
  pw:browser [pid=794121][err] #26 0x55b5259ae309 ChromeMain +10ms
  pw:browser [pid=794121][err] #27 0x7fe52acdb850 (/usr/lib/libc.so.6+0x2384f) +0ms
  pw:browser [pid=794121][err] #28 0x7fe52acdb90a __libc_start_main +0ms
  pw:browser [pid=794121][err] #29 0x55b5259ae02a _start +10ms
  pw:browser [pid=794121][err]   r8: 0000000000000002  r9: 0000000000000000 r10: 000055b5307d7b80 r11: 00007fe529b6e988 +0ms
  pw:browser [pid=794121][err]  r12: 000055b53071e000 r13: 000007fc00f4e200 r14: 000007fc00f4e1c0 r15: 000007fc00f4e1d8 +0ms
  pw:browser [pid=794121][err]   di: 0000000000000148  si: 000007fc00f4e1c0  bp: 00007ffc13df7700  bx: 0000000000000148 +0ms
  pw:browser [pid=794121][err]   dx: 000055b523accb20  ax: 0000000000000000  cx: 0000000030127401  sp: 00007ffc13df76b0 +0ms
  pw:browser [pid=794121][err]   ip: 000055b525b5d670 efl: 0000000000010206 cgf: 002b000000000033 erf: 0000000000000004 +0ms
  pw:browser [pid=794121][err]  trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000148 +1ms
  pw:browser [pid=794121][err] [end of stack trace] +0ms
  pw:browser [pid=794121][err] libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null) +41ms
[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser [pid=794122][err] libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null) +103ms
[setup] › auth.setup.ts:10:6 › authenticate as admin
  pw:browser [pid=794121] <process did exit: exitCode=null, signal=SIGSEGV> +465ms
  pw:browser [pid=794121] starting temporary directories cleanup +0ms
  pw:api <= browserContext.newPage failed +737ms
  1) [setup] › auth.setup.ts:10:6 › authenticate as admin ──────────────────

    Error: Page closed




[setup] › auth.setup.ts:3:6 › authenticate as user
  pw:browser [pid=794122] <process did exit: exitCode=null, signal=SIGSEGV> +489ms
  pw:browser [pid=794122] starting temporary directories cleanup +0ms
  pw:api <= browserContext.newPage failed +781ms
  2) [setup] › auth.setup.ts:3:6 › authenticate as user ────────────────────────────────────────────

    Error: Page closed




  2 failed
    [setup] › auth.setup.ts:3:6 › authenticate as user ─────────────────────────────────────────────
    [setup] › auth.setup.ts:10:6 › authenticate as admin ───────────────────────────────────────────
  2 skipped

To open last HTML report run:

  npx playwright show-report

 ELIFECYCLE  Command failed with exit code 1.
```

**Expected**

Auth setup tests should run in headed mode and not skip all other tests.

**Actual**

Auth setup tests fails and skips other tests in headed mode. Currently, it also prevents tester from running tests in debug mode as it tries to open browser for auth setup tests.

**Comment by deepanchal** [comment]
TLDR; Issues with chromium were causing errors when running auth setup tests

I was having issues running tests with chromium. I just realized that the setup step uses `chromium` to run auth setup and save browser state. Since my chromium wasn't working, it wasn't able to run those auth setup tests. I made the following change in my repo which uses firefox for running setup tests and it works as expected.

After making this change

```diff
diff --git a/playwright.config.ts b/playwright.config.ts
index 702fda1..cc2cda6 100644
--- a/playwright.config.ts
+++ b/playwright.config.ts
@@ -35,6 +35,7 @@ export default defineConfig({
     // Setup project
     {
       name: 'setup',
+      use: { ...devices['Desktop Firefox'] },
       testMatch: /.*\.setup\.ts/,
     },
```

headed mode works fine `npm exec -- playwright test --headed`

I am sorry for any confusion

**Comment by deepanchal** [comment]
Running `DEBUG='pw:api,pw:browser*' npx playwright cr 'https://google.com'` gives the following error. Therefore, this is the actual issue I am having. I will close this issue now as it's caused by some other issue I am having with chromium.

```log
  pw:api => qr.parse started +0ms
  pw:browser <launching> /home/deep/.cache/ms-playwright/chromium-1071/chrome-linux/chrome --disable-field-trial-config --disable-background-networking --enable-features=NetworkService,NetworkServiceInProcess --disable-background-timer-throttling --disable-backgrounding-occluded-windows --disable-back-forward-cache --disable-breakpad --disable-client-side-phishing-detection --disable-component-extensions-with-background-pages --disable-component-update --no-default-browser-check --disable-default-apps --disable-dev-shm-usage --disable-extensions --disable-features=ImprovedCookieControls,LazyFrameLoading,GlobalMediaControls,DestroyProfileOnBrowserClose,MediaRouter,DialMediaRouteProvider,AcceptCHFrame,AutoExpandDetailsElement,CertificateTransparencyComponentUpdater,AvoidUnnecessaryBeforeUnloadCheckSync,Translate --allow-pre-commit-input --disable-hang-monitor --disable-ipc-flooding-protection --disable-popup-blocking --disable-prompt-on-repost --disable-renderer-backgrounding --force-color-profile=srgb --metrics-recording-only --no-first-run --enable-automation --password-store=basic --use-mock-keychain --no-service-autorun --export-tagged-pdf --no-sandbox --user-data-dir=/tmp/playwright_chromiumdev_profile-3epAeh --remote-debugging-pipe --no-startup-window +0ms
  pw:browser <launched> pid=865796 +5ms
  pw:browser [pid=865796][err] Gtk-Message: 14:41:02.333: Failed to load module "appmenu-gtk-module" +136ms
  pw:browser [pid=865796][err] [865796:865796:0718/144102.410863:ERROR:chrome_browser_cloud_management_controller.cc(162)] Cloud management controller initialization aborted as CBCM is not enabled. +78ms
  pw:api <= qr.parse succeeded +333ms
  pw:api =>  started +1ms
  pw:api <=  succeeded +8ms
  pw:api =>  started +2ms
  pw:browser <launching> /home/deep/.cache/ms-playwright/chromium-1071/chrome-linux/chrome --disable-field-trial-config --disable-background-networking --enable-features=NetworkService,NetworkServiceInProcess --disable-background-timer-throttling --disable-backgrounding-occluded-windows --disable-back-forward-cache --disable-breakpad --disable-client-side-phishing-detection --disable-component-extensions-with-background-pages --disable-component-update --no-default-browser-check --disable-default-apps --disable-dev-shm-usage --disable-extensions --disable-features=ImprovedCookieControls,LazyFrameLoading,GlobalMediaControls,DestroyProfileOnBrowserClose,MediaRouter,DialMediaRouteProvider,AcceptCHFrame,AutoExpandDetailsElement,CertificateTransparencyComponentUpdater,AvoidUnnecessaryBeforeUnloadCheckSync,Translate --allow-pre-commit-input --disable-hang-monitor --disable-ipc-flooding-protection --disable-popup-blocking --disable-prompt-on-repost --disable-renderer-backgrounding --force-color-profile=srgb --metrics-recording-only --no-first-run --password-store=basic --use-mock-keychain --no-service-autorun --export-tagged-pdf --no-sandbox --app=data:text/html, --window-size=600,600 --window-position=1020,10 --test-type= --user-data-dir=/tmp/playwright_chromiumdev_profile-cosEn8 --remote-debugging-pipe about:blank +161ms
  pw:browser <launched> pid=865939 +9ms
  pw:browser [pid=865796][err] libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null) +12ms
  pw:browser [pid=865939][err] Gtk-Message: 14:41:02.722: Failed to load module "appmenu-gtk-module" +129ms
  pw:browser [pid=865939][err] [865939:865939:0718/144102.778091:ERROR:chrome_browser_cloud_management_controller.cc(162)] Cloud management controller initialization aborted as CBCM is not enabled. +56ms
  pw:browser [pid=865939][err] libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null) +219ms
  pw:api   "commit" event fired +624ms
  pw:api   "commit" event fired +8ms
  pw:api   navigated to "data:text/html," +0ms
  pw:api   "load" event fired +2ms
  pw:api   "domcontentloaded" event fired +0ms
  pw:api navigating to "https://playwright/index.html", waiting until "load" +28ms
  pw:api   "commit" event fired +27ms
  pw:api   navigated to "https://playwright/index.html" +0ms
  pw:api   "domcontentloaded" event fired +77ms
  pw:api   "load" event fired +6ms
  pw:api <=  succeeded +7ms
  pw:api =>  started +0ms
  pw:browser [pid=865796][err] [0718/144103.295812:ERROR:elf_dynamic_array_reader.h(64)] tag not found +299ms
  pw:browser [pid=865796][err] [0718/144103.296511:ERROR:elf_dynamic_array_reader.h(64)] tag not found +0ms
  pw:browser [pid=865796][err] [0718/144103.296615:ERROR:elf_dynamic_array_reader.h(64)] tag not found +0ms
  pw:browser [pid=865796][err] [0718/144103.296770:ERROR:elf_dynamic_array_reader.h(64)] tag not found +0ms
  pw:browser [pid=865796][err] [0718/144103.299450:ERROR:file_io_posix.cc(144)] open /sys/devices/system/cpu/cpu0/cpufreq/scaling_cur_freq: No such file or directory (2) +3ms
  pw:browser [pid=865796][err] [0718/144103.299482:ERROR:file_io_posix.cc(144)] open /sys/devices/system/cpu/cpu0/cpufreq/scaling_max_freq: No such file or directory (2) +0ms
  pw:browser [pid=865796][err] Received signal 11 SEGV_MAPERR 000000000148 +6ms
  pw:browser [pid=865796][err] #0 0x55d2cdf71302 base::debug::CollectStackTrace() +5ms
  pw:browser [pid=865796][err] #1 0x55d2cdf5e753 base::debug::StackTrace::StackTrace() +6ms
  pw:browser [pid=865796][err] #2 0x55d2cdf70d91 base::debug::(anonymous namespace)::StackDumpSignalHandler() +5ms
  pw:browser [pid=865796][err] #3 0x7fe19903aab0 (/usr/lib/libc.so.6+0x39aaf) +0ms
  pw:browser [pid=865796][err] #4 0x55d2c9cdd670 base::ObserverList<>::AddObserver() +0ms
  pw:browser [pid=865796][err] #5 0x55d2d37ad375 AvatarMenu::AvatarMenu() +12ms
  pw:browser [pid=865796][err] #6 0x55d2d37a9dff DbusAppmenu::Initialize() +13ms
  pw:browser [pid=865796][err] #7 0x55d2d37ac296 DbusAppmenuRegistrar::InitializeMenu() +12ms
  pw:browser [pid=865796][err] #8 0x55d2d37acaa3 DbusAppmenuRegistrar::OnNameOwnerChanged() +13ms
  pw:browser [pid=865796][err] #9 0x55d2c9cd04f1 base::internal::Invoker<>::Run() +0ms
  pw:browser [pid=865796][err] #10 0x55d2ca4c4911 base::internal::Invoker<>::RunOnce() +0ms
  pw:browser [pid=865796][err] #11 0x55d2cdf011c4 base::TaskAnnotator::RunTaskImpl() +5ms
  pw:browser [pid=865796][err] #12 0x55d2cdf1b52f base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWorkImpl() +6ms
  pw:browser [pid=865796][err] #13 0x55d2cdf1afa9 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWork() +5ms
  pw:browser [pid=865796][err] #14 0x55d2cdf1ba05 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWork() +5ms
  pw:browser [pid=865796][err] #15 0x55d2cdf84e14 base::MessagePumpGlib::Run() +4ms
  pw:browser [pid=865796][err] #16 0x55d2cdf1bd79 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::Run() +5ms
  pw:browser [pid=865796][err] #17 0x55d2cdee1e66 base::RunLoop::Run() +5ms
  pw:browser [pid=865796][err] #18 0x55d2cbec4fa3 content::BrowserMainLoop::RunMainMessageLoop() +2ms
  pw:browser [pid=865796][err] #19 0x55d2cbec6aa2 content::BrowserMainRunnerImpl::Run() +2ms
  pw:browser [pid=865796][err] #20 0x55d2cbec2616 content::BrowserMain() +1ms
  pw:browser [pid=865796][err] #21 0x55d2cd367088 content::RunBrowserProcessMain() +4ms
  pw:browser [pid=865796][err] #22 0x55d2cd3689e4 content::ContentMainRunnerImpl::RunBrowser() +4ms
  pw:browser [pid=865796][err] #23 0x55d2cd3687d6 content::ContentMainRunnerImpl::Run() +3ms
  pw:browser [pid=865796][err] #24 0x55d2cd365d90 content::RunContentProcess() +4ms
  pw:browser [pid=865796][err] #25 0x55d2cd365fbd content::ContentMain() +3ms
  pw:browser [pid=865796][err] #26 0x55d2c9b2e309 ChromeMain +11ms
  pw:browser [pid=865796][err] #27 0x7fe199024850 (/usr/lib/libc.so.6+0x2384f) +0ms
  pw:browser [pid=865796][err] #28 0x7fe19902490a __libc_start_main +0ms
  pw:browser [pid=865796][err] #29 0x55d2c9b2e02a _start +10ms
  pw:browser [pid=865796][err]   r8: 0000000000000000  r9: 00002d18016e10e0 r10: 000055d2d4957b80 r11: 00007fe197f53988 +0ms
  pw:browser [pid=865796][err]  r12: 000055d2d489e000 r13: 00002d18016e11c0 r14: 00002d18016e1180 r15: 00002d18016e1198 +0ms
  pw:browser [pid=865796][err]   di: 0000000000000148  si: 00002d18016e1180  bp: 00007ffe68ce80c0  bx: 0000000000000148 +0ms
  pw:browser [pid=865796][err]   dx: 000055d2c7c4cb20  ax: 0000000000000000  cx: 00000000d42a7401  sp: 00007ffe68ce8070 +0ms
  pw:browser [pid=865796][err]   ip: 000055d2c9cdd670 efl: 0000000000010202 cgf: 002b000000000033 erf: 0000000000000004 +0ms
  pw:browser [pid=865796][err]  trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000148 +0ms
  pw:browser [pid=865796][err] [end of stack trace] +0ms
  pw:api   "networkidle" event fired +505ms
  pw:browser [pid=865939] <gracefully close start> +647ms
  pw:browser [pid=865796] <kill> +1ms
  pw:browser [pid=865796] <will force kill> +0ms
  pw:browser [pid=865796] starting temporary directories cleanup +0ms
  pw:browser [pid=865796] finished temporary directories cleanup +93ms
  pw:browser [pid=865939] <kill> +1ms
  pw:browser [pid=865939] <will force kill> +0ms
  pw:browser [pid=865939] starting temporary directories cleanup +0ms
  pw:browser [pid=865939] finished temporary directories cleanup +89ms
/tmp/playwright-auth-setup-tests-bug/node_modules/playwright-core/lib/server/chromium/crPage.js:377
      this._firstNonInitialNavigationCommittedReject(new Error('Page closed'));
                                                     ^

Error: Page closed
    at CRSession.<anonymous> (/tmp/playwright-auth-setup-tests-bug/node_modules/playwright-core/lib/server/chromium/crPage.js:377:54)
    at Object.onceWrapper (node:events:627:28)
    at CRSession.emit (node:events:525:35)
    at /tmp/playwright-auth-setup-tests-bug/node_modules/playwright-core/lib/server/chromium/crConnection.js:211:39

Node.js v18.16.1
```
